### PR TITLE
Fix clap

### PR DESCRIPTION
**Pull request by Eskaan**
Requested pr on your fork

### CODE REVIEW

**Comment by Eskaan** [comment]
Force-push to make co-authorship work.

File: leftwm/src/bin/leftwm-check.rs [hunk 1]
```diff
@@ -10,41 +10,21 @@ use std::path::Path;
...
-                .help("Outputs received configuration file.")
-        )
-        .arg(
-            Arg::new(MIGRATE_ARG)
-                .short('m')
-                .long("migrate-toml-to-ron")
-                .help("Migrates an exesting `toml` based config to a `ron` based one.\nKeeps the old file for reference, please delete it manually.")
-        )
+        .args(&[
+            arg!(-v --verbose "Outputs received configuration file."),
```
Comment (VuiMuich) [inline leftwm/src/bin/leftwm-check.rs:20]: Couldn't we also do:
```suggestion
            arg!(verbose: -v "Outputs received configuration file."),
```

Comment (Eskaan) [reply leftwm/src/bin/leftwm-check.rs:20]: Yes, but then you could only use -v and not --verbose on the command line.

Comment (VuiMuich) [reply leftwm/src/bin/leftwm-check.rs:20]: I see, I thought, the syntax of `migrate` would result in the three options `-m, --migrate, --migrate-toml-to-ron`, but that obviously was a missconception then.

**Pull request merged**